engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -1282,6 +1267,19 @@ namespace dmGameObject
             return false;
         }
 
+        // Update the transform for all parent-less objects
+        for (uint32_t i=0;i!=new_instances.Size();i++)
+        {
+            if (!GetParent(new_instances[i]))
+            {
+                new_instances[i]->m_Transform = dmTransform::Mul(transform, new_instances[i]->m_Transform);
+                dmLogInfo("CollectionSpawnFromDescInternal new_instances scale x: %f y: %f z: %f", new_instances[i]->m_Transform.GetScale().getX(), new_instances[i]->m_Transform.GetScale().getY(), new_instances[i]->m_Transform.GetScale().getZ());
```

Comment:
This log info feels a bit redundant imho.
If some issue pops up, we can debug it anyways.

Reply:
Oh, that's my debugging log! I will remove it!